enhancements/helm3/console.md

```diff
@@ -0,0 +1,146 @@
+   * The content of the chart repository could be crawled and served using a (containerized or external) HTTP server, e.g. nginx
+
+2. The URL serving the above static content would need to be passed to chart repository proxy running inside the cluster. 
+
+### Configuring Helm Chart Repository location
+
+Configuring Helm repository location could be modeled similar to [`OperatorSource`](https://github.com/operator-framework/operator-marketplace/blob/7d230952a1045624b7601b4d6e1d45b3def4cf76/deploy/crds/operators_v1_operatorsource_crd.yaml). The resource would need to contain the following field:
+
+```yaml
+url: http://my.chart-repo.org/stable
+
+# optional and only needed for UI purposes
+displayName: myChartRepo
+
+# optional and only needed for UI purposes
+description: my private chart repo
+
+# set to true if need to be disabled temporarly
+disabled: true
+```
+
+Due to future federated usecases ([ODC-2994](https://issues.redhat.com/browse/ODC-2994))), a cluster admin should be able to declare multiple chart repositories.
+Such list could be embedded (cluster-wide, top-level `Helm` resource)[https://github.com/openshift/api/tree/master/config/v1]:
+
+```yaml
+apiVersion: config.openshift.io/v1
+kind: Helm
+metadata:
+  name: cluster
+spec:
+  chartRepositories:
+    - url: http://my.chart-repo.org/stable
+      displayName: myChartRepo
+      description: my private chart repo
+
+
+```
+
+The console operator would watch for changes on it and reconfigure the chart repository proxy. The console backend already implements a few helm endpoints (including the chart proxy), but the future might require to extract them into a separate service (e.g. to make openshift more modular). Hence, it would be good to define the configuration as the top-level resource detaching API from the implementation. Cluster admin would easily discover the new resource and manage it either via CLI or through UI. 
+
+![Helm Cluster Configuration](assets/openshift-administration-cluster-settings.png)
+
+#### Alternatives
+
+1. The configuration could be embedded into cluster-wide [`Console` config](https://github.com/openshift/api/blob/master/config/v1/types_console.go#L26)
+
+Such approach would hide a bit the configuration, and make its discovery by admins harder. It becomes closely coupled with the console. Extracting Helm endpoints into a separate service would require moving the config as well.
+
+2. The configuration could be embedded into [`Console` operator config](https://github.com/openshift/api/blob/master/operator/v1/types_console.go#L26)
+
+Such approach shares the similar issues with the previous alternative. Discovering the config by cluster admins is even harder, it would live in `openshift-console-operator` namespace.
+
+3. OLM operator for Helm endpoints. Helm configuration would move to the top-level resource inside `openshift-helm` namespace, along with the service implementing the endpoints.
+
```

Comment:
I think all operators from `OLM` are installed via `openshift-operators`, though perhaps this is configurable.

Reply:
they are not.  global operators, by default, are installed there, but even that is just a default not a requirement.

single namespace operators are installed in the namespace the admin wants them in.